### ISSUE TITLE
Install fails via Yarn due to unresolved dependency

### ISSUE THREAD

**Issue by chaosmail** [monounity/karma-typescript#233]
Steps to reproduce:

````
$ yarn add karma-typescript
yarn add v1.3.2
[1/4] Resolving packages...
[2/4] Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/pinkie/-/pinkie-2.0.5.tgz: Request failed \"404 Not Found\"".
info If you think this is a bug, please open a bug report with the information provided in "/home/ckoerner/software/deeplearnjs/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by erikbarke** [comment]
This seems to be part of a bigger yarn/npm/pinkie issue: https://github.com/floatdrop/pinkie/issues/18!

**Comment by robwalch** [comment]
I was having this problem because of yarn.lock file I created for a project on Saturday during the npm outage.

Removing the yarn.lock file and re-running `yarn install` to recreate it fixed the issue (pinkie-2.0.4 is installed).
